media_service/models.py

```diff
@@ -138,6 +138,10 @@ class Meta:
     def __unicode__(self):
         return u'{0}:{1}:{2}'.format(self.id, self.lti_context_id, self.title)
 
+
+def metadata_default():
```

Comment:
You're right, it should probably be a static method on `Resource` since it's specific to just that model and would make the unittest clearer. I'll make that change.